PyComServer/PyComServer.pyproj

```diff
@@ -0,0 +1,54 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>b48dc892-e120-49f5-8e20-d497214811d5</ProjectGuid>
+    <ProjectHome>.</ProjectHome>
+    <StartupFile>PyComServer.py</StartupFile>
+    <SearchPath>
+    </SearchPath>
+    <WorkingDirectory>.</WorkingDirectory>
+    <OutputPath>.</OutputPath>
+    <Name>PyComServer</Name>
+    <RootNamespace>PyComServer</RootNamespace>
+    <LaunchProvider>Standard Python launcher</LaunchProvider>
+    <CommandLineArguments>/regserver</CommandLineArguments>
+    <EnableNativeCodeDebugging>False</EnableNativeCodeDebugging>
+    <InterpreterId>MSBuild|env|$(MSBuildProjectFullPath)</InterpreterId>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DebugSymbols>true</DebugSymbols>
+    <EnableUnmanagedDebugging>false</EnableUnmanagedDebugging>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DebugSymbols>true</DebugSymbols>
+    <EnableUnmanagedDebugging>false</EnableUnmanagedDebugging>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="PyComServer.py" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="RoyalPython.idl" />
```

Comment:
Could be nice to see the registration bat script here also.

Reply:
Unfortunately I did not manage to find beautiful in .pyproj file, because [Exec](https://docs.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022) task use regular cmd instead of developer command prompt and I can not find midl compiler.